### PR TITLE
CRAYSAT-1923: Update `sat bootsys` to support CFS v2 or v3

### PR DESCRIPTION
**Pull request by annapoorna-s-alt**
## Summary and Scope

_Update `sat bootsys` to support CFS v2 or v3_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CRAYSAT-1923](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-1923)
* Resolves [CRAYSAT-1847](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-1847)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  fanta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

Test `sat bootsys` for both cfs v2 and cfs v3 version including the paging property

## Risks and Mitigations

_Low_

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Comment by annapoorna-s-alt** [comment]
Testing output - https://gist.github.com/annapoorna-s-alt/adc093e8940f4d761dde034c4c895194

File: docs/man/sat-bootsys.8.rst [hunk 1]
```diff
@@ -159,6 +159,9 @@ These options apply to both the ``shutdown`` and ``boot`` actions.
 **--bos-version BOS_VERSION**
         The version of the BOS API to use when launching BOS sessions.
 
+**--cfs-version CFS_VERSION**
+        The version of the CFS API to use when launching CFS sessions.
```
Comment (haasken-hpe) [inline docs/man/sat-bootsys.8.rst:163]: We only query the CFS API for active sessions. 

```suggestion
        The version of the CFS API to use when querying CFS sessions.
```

We don't launch any CFS sessions in `sat bootsys` like we do for BOS sessions in the `bos-operations` stage. Perhaps the documentation for `--bos-version` above would more accurately be "The version of the BOS API to use when launching or querying BOS sessions."

**Review by haasken-hpe — APPROVED** [review]
Minor comment. Looks good!

File: docs/man/sat-bootsys.8.rst (hunk 1 above)
Comment (annapoorna-s-alt) [reply docs/man/sat-bootsys.8.rst]: Addressed

**Pull request merged**